### ISSUE TITLE
Inbox should stay on current tab

### ISSUE THREAD

**Issue by grahamsinger** [tsgrp/hpi#1025]
See screenshot:

![tabsondashlet](https://cloud.githubusercontent.com/assets/8784597/13303012/740215ea-db12-11e5-9697-7321b99fb816.gif)

**Comment by gsteimer** [comment]
We coded it this way so that you can still see the task once you acquire it (since the task moves to the My Tasks tab).

I can see users wanting it both ways:

- A user who wants to acquire a bunch of tasks one after the other probably doesn't want the tab to switch
- A user who wants to acquire a task and then work on it right away probably _does_ want the tab to switch.

Either way, we should always switch when acquiring the last task in a group list.  Otherwise, the user is left on a blank tab.

**Comment by grahamsinger** [comment]
Yeah the only reason I opened this was that it actually looks like in `inbox.js` there is code that is trying to keep the tab on the active one.  But what you said also makes sense.
